images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -35,5 +35,12 @@ Choco-Install -PackageName cabal
 Invoke-PesterTests -TestFile 'Haskell'
 
 # install minimal ghcup, utilizing pre-installed msys2 at C:\msys64
-$bootstrapHaskell = Invoke-WebRequest https://www.haskell.org/ghcup/sh/bootstrap-haskell.ps1 -UseBasicParsing
-Invoke-Command -ScriptBlock ([ScriptBlock]::Create($bootstrapHaskell)) -ArgumentList $false, $true, $true, $false, $false, $false, $false, C:\, "", C:\msys64, C:\cabal
+$msysPath = "C:\msys64"
+$ghcupPrefix = "C:\"
+$cabalDir = "C:\cabal"
+Invoke-Command -ScriptBlock ([ScriptBlock]::Create($bootstrapHaskell)) -ArgumentList $false, $true, $true, $false, $false, $false, $false, $ghcupPrefix, "", $msysPath, $cabalDir
```

Comment:
@hasufell, This part of script is missing -> `ScriptBlock ([ScriptBlock]::Create($bootstrapHaskell))`

Reply:
@al-cheb done